email-templates/task1.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>first email template</title>
+</head>
+<body style="font-family: Arial, sans-serif; text-align: center; background-color: #e4f3ea; margin: 0px auto; max-width: 600px; padding: 30px; border-radius: 8px; box-shadow: 0 0 10px rgba(229, 205, 205, 0.1);">
+
+  <h1 style="color: #0074cc;">HAPPY NEW YEAR</h1>
+
+  <p style="color: #333;">
+    Dear Ronaldo,
+    
+  </p>
+
+  <p style="color: #333;">
+    Wishing you joy, success, and fulfillment in the coming year. May each day bring new opportunities and cherished moments. .
+  </p>
+
+  <p style="color: hsl(89, 51%, 46%);">
```

Comment:
Instead of using `hsl` use hex values for colors. For example, for white use `#ffffff`

Reply:
Used hex values and updated